src/main/scala/mesosphere/marathon/storage/StorageConf.scala

```diff
@@ -34,6 +34,12 @@ trait StorageConf extends ZookeeperConf with BackupConf {
     default = Some(32)
   )
 
+  lazy val groupVersionsCacheSize = opt[Int](
+    "group_versions_cache_size",
+    descr = "Size of the groups repository versions cache.",
+    default = Some(1000)
```

Comment:
How does this default correspond to the above-mentioned formula? Why is it `100`?

Reply:
backwards-compatibility

Reply:
But that value doesn't make any sense since we cap it earlier now?

Reply:
Ok, after discussing it in person, that makes sense for the backward compatibility.

Reply:
Could you share the answer here?

Reply:
Basically, one can already have 999 groups there which are all used by some deployments. If you change them to 100 here, the GCActor will start to work constantly. Hence, it should stay 1000, at least for now and might get a more sensible default in the next version.